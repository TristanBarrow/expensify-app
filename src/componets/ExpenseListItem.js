import React from 'react';
import { Link } from 'react-router-dom'; 

const ExpenseListItem = ({ description, amount, createdAt, id }) => (
  <div>
    <Link to={`/edit/${id}`}>    
      <h3>{description}</h3>
    </Link>
    <p>
      Amount: {amount} <br />
      Time: {createdAt}
    </p>
  </div>
);

export default ExpenseListItem;