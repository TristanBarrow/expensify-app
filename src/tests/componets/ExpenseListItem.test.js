 import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../componets/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('render expense list item', () => {
  const wrapper = shallow(<ExpenseListItem { ...expenses[1] }/>);
  expect(wrapper).toMatchSnapshot();
});