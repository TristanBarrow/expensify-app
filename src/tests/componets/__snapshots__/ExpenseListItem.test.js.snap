// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render expense list item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/ID 2"
  >
    <h3>
      ABC
    </h3>
  </Link>
  <p>
    Amount: 
    300
     
    <br />
    Time: 
    0
  </p>
</div>
`;
